### PR TITLE
fix: add collisionBoundary for HoverCard Content to avoid flicker problem

### PR DESCRIPTION
**Pull request by aiyogg**
fix: #32

### CODE REVIEW

**Review by CaliCastle — APPROVED** [review]
Thank you so much ❤️

File: components/links/PeekabooLink.tsx [hunk 1]
```diff
@@ -73,7 +72,10 @@ export function PeekabooLink({
       <AnimatePresence mode="wait">
         {isOpen && (
           <HoverCard.Portal forceMount>
-            <HoverCard.Content asChild>
+            <HoverCard.Content
+              asChild
+              collisionBoundary={250}
```
Comment (CaliCastle) [inline components/links/PeekabooLink.tsx:77]:
```suggestion
              collisionBoundary={[250]}
```

**Review by CaliCastle — CHANGES_REQUESTED** [review]
Looks like TypeScript is complaining about the type here

File: components/links/PeekabooLink.tsx (hunk 1 above)
Comment (aiyogg) [reply components/links/PeekabooLink.tsx]: You're right, this is my negligence.

File: components/links/PeekabooLink.tsx [hunk 2]
```diff
@@ -73,7 +72,10 @@ export function PeekabooLink({
       <AnimatePresence mode="wait">
         {isOpen && (
           <HoverCard.Portal forceMount>
-            <HoverCard.Content asChild>
+            <HoverCard.Content
+              asChild
+              collisionBoundary={[250]}
```
Comment (CaliCastle) [inline components/links/PeekabooLink.tsx:77]:
```suggestion
              collisionPadding={250}
```
I think it should be padding instead, right?
https://www.radix-ui.com/primitives/docs/components/hover-card

**Review by CaliCastle — CHANGES_REQUESTED** [review]

File: components/links/PeekabooLink.tsx (hunk 2 above)
Comment (aiyogg) [reply components/links/PeekabooLink.tsx]: Sorry, I also realized after resolving the conversation, that `collisionPadding ` is correct. 😂

**Review by CaliCastle — APPROVED** [review]

**Pull request merged**